tests/views_integration/Tribe/Events/Views/V2/Views/Day_ViewTest.php

```diff
@@ -22,6 +22,9 @@ public function setUp() {
 		// Remove v1 filtering to have consistent results.
 		remove_filter( 'tribe_events_before_html', [ TEC::instance(), 'before_html_data_wrapper' ] );
 		remove_filter( 'tribe_events_after_html', [ TEC::instance(), 'after_html_data_wrapper' ] );
+
+		// Let's make sure `$_SERVER['REQUEST_URI']` is reset, Views will set it in the `setup_the_loop` method.
+		$_SERVER['REQUEST_URI'] = '';
```

Comment:
Do you want to include this in the PR? or remove for now and add to the branch that fixes these tests?

Reply:
Yes, thanks.